CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ authentication is still required for an email notifier, but the user can now cho
 ### Technical Changes
 - ROX-11181: Any clusters that have been unhealthy (defined as central being unable to reach sensor running on those clusters) for a period of time will be automatically removed. By default, it will remove if it's been unhealthy for 90 days, however that can be configured in the System Configuration page or using the cluster API.
   - Any cluster that is expected to be unavailable for a period of time (e.g. clusters used in disaster recovery), can be tagged with a customizable label. Clusters with those labels will never be removed automatically.
+- ROX-7591: Policy `Fixable CVSS >= 6 and Privileged` disabled by default, new policy `Privileged Container(s) with Important and Critical CVE(s)` added and enabled by default.
```

Comment:
The policy is disabled by default only on new installations because we are not updating existing system policies.